api/serializers/__init__.py

```diff
@@ -13,13 +13,14 @@
 from .token_value import TokenValueSerializer
 from .user_consent import UserConsentSerializer
 from .user_stats import UserStatsSerializer
+from .list_user import UsersCountSerializers
```

Comment:
Remove this and other unnecessary declarations, and instead collate most things to admin viewset